python_modules/dagster/dagster/_cli/asset.py

```diff
@@ -158,3 +158,61 @@ def asset_wipe_command(key, **cli_args):
             click.echo("Removed asset indexes from event logs")
         else:
             click.echo("Exiting without removing asset indexes")
+
+
+@asset_cli.command(name="wipe-cache")
```

Comment:
I think it's worth being verbose and specific with the name - e.g. `wipe-partition-status-cache`.